deployment/playbook.yaml

```diff
@@ -14,7 +14,9 @@
   roles:
   - role: 'grycap.docker'
     docker_config_values: { "mtu": 1400 }
-
+  - role: 'grycap.motley-cue'
+    ssh_oidc_my_vo: false
+    ssh_oidc_other_vos: cloud.egi.eu
```

Comment:
can we be more specific on the role from the VO?

Reply:
Checking: https://github.com/fedcloud-eu/vault-doc/issues/2

Reply:
is that issue related to the role restriction in motley-cue? I understand this is independent to secrets but a matter of how the entitlement is configured ssh-oidc

Reply:
Yes, the variable `ssh_oidc_other_vos: cloud.egi.eu` in the Ansible role above will be translated into:
```
authorised_vos = ['urn:mace:egi.eu:group:cloud.egi.eu:role=member#aai.egi.eu']
```
inside a file `/etc/motley_cue/motley_cue.conf` on the target VM via:
* https://github.com/marcvs/contextualise_ssh_server/blob/v0.8.6/contextualise_ssh_server/motley_cue.template.conf#L127
* https://github.com/grycap/ansible-role-motley-cue/blob/950d21af0724550dc17dd8019165ca2a4e2a7b93/tasks/main.yml#L101-L119
* https://github.com/grycap/ansible-role-motley-cue/blob/950d21af0724550dc17dd8019165ca2a4e2a7b93/tasks/main.yml#L79-L82

The solution is to configure `/etc/motley_cue/motley_cue.conf` on the target VM with this instead:
```
authorised_vos = ['urn:mace:egi.eu:group:cloud.egi.eu:role=auditor#aai.egi.eu']
```

I think the best way to accomplish this is to update this task in Ansible:
* https://github.com/grycap/ansible-role-motley-cue/blob/950d21af0724550dc17dd8019165ca2a4e2a7b93/tasks/main.yml#L79-L82

To make the `role=member#aai.egi.eu` part of the entitlement configurable instead of hard coded.

What do you think?

If you agree, I will open a PR in https://github.com/grycap/ansible-role-motley-cue/

Reply:
Here it is: https://github.com/grycap/ansible-role-motley-cue/pull/13

Reply:
Ok, I have now applied the changes required:

1. restrict access to members of the `cloud.egi.eu` VO with the `auditor` role.
2. once logged in, these members can use unrestricted `sudo`

Please let me know your thoughts.